packages/console/src/components/DetailsForm/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { forwardRef } from 'react';
+import type { ForwardedRef, HTMLProps, ReactNode } from 'react';
+
+import SubmitFormChangesActionBar from '../SubmitFormChangesActionBar';
+import * as styles from './index.module.scss';
+
+type Props = HTMLProps<HTMLFormElement> & {
+  isDirty: boolean;
+  isSubmitting: boolean;
+  children: ReactNode;
+  onDiscard: () => void;
+};
+
+const DetailsForm = (
+  { isDirty, isSubmitting, onDiscard, children, ...rest }: Props,
+  reference: ForwardedRef<HTMLFormElement>
+) => {
+  return (
+    <form {...rest} ref={reference} className={styles.container}>
+      <div className={styles.fields}>{children}</div>
```

Comment:
nit: instead of adding an extra wrapper layer prefer adding a empty placeholder child element with flex: 1

like 

```
<form>
{children}
<div style={{flex: 1}} />
<SubmitFormChangesActionBar />
</form>
```

Reply:
Or maybe 
```
<div container>
<form>
{children}
</form>
<SubmitFormChangesActionBar />
</form>
</div>
```

better keep the form's children directly representing the input fields.

Reply:
Good idea, but this component aims at sharing the same styles and logic between every detail page, not a commonly used form wrapper.